docling/models/genos_dots_ocr_layout_model.py

```
@@ -728,6 +727,46 @@ def _process_page(self, conv_res: ConversionResult, page: Page) -> Page:
+                if label in self.TABLE_LABELS:
+                    table_htmls = (
+                        _extract_all_table_html(pred_item_text)
+                        if pred_item_text
+                        else []
+                    )
+                    if len(table_htmls) > 1:
+                        sub_bboxes = _split_bbox_vertically(
+                            bbox_values, len(table_htmls)
+                        )
+                        for band_idx, (sub_bbox, table_html) in enumerate(
+                            zip(sub_bboxes, table_htmls)
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)